workers/notification/lib/notification/notificationworker.coffee

```diff
@@ -0,0 +1,335 @@
+cors = require 'cors'
+http = require 'http'
+amqp = require 'amqp'
+
+sockjs = require 'sockjs'
+helmet = require 'helmet'
+
+express    = require 'express'
+{ argv }   = require 'optimist'
+bodyParser = require 'body-parser'
+
+KONFIG = require 'koding-config-manager'
+bongo  = require '../../../../servers/lib/server/bongo'
+
+module.exports = class NotificationWorker
+
+
+  QUEUEPREFIX  = 'NodeNotification'
+  QUEUEOPTIONS = { durable: yes, autoDelete: yes }
+
+  EXCHANGENAME = 'NotificationMessageBus:0'
+  EXCHANGE_OPTIONS = {
+    autoDelete: yes
+    durable: yes
+    type: 'fanout'
+  }
+
+  [ NOTREADY, READY ] = [ 0, 1 ]
+
+
+  constructor: ->
+
+    @_state = NOTREADY
+    @_queue = []
+
+    @_queueName = "#{QUEUEPREFIX}:#{Math.random().toString().slice(2, 10)}"
+
+    @_connections = {}
+    @_verifiedConnections = []
+    @_verifiedRoutes = []
+
+
+  start: ->
+
+    @log 'starting worker...'
+
+    @_connectToRabbitMQ()
```

Comment:
we're consuming but we're not sending them through before making the bongo connection https://github.com/koding/koding/pull/11234/files#diff-8367197550fb4e5f4852ba02179f6710R70